src/.vscode/tasks.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I don't want this file.

Reply:
you are right....I forgot gitignore sorry